package.json

```diff
@@ -18,6 +18,9 @@
   },
   "homepage": "https://github.com/egnyte/wopi-proof-validator#readme",
   "dependencies": {
-    "bignum": "0.13.0"
+    "cross-env": "^6.0.3",
```

Comment:
cross-env seems like a dev dependency. Should it be in devDependencies section? 

I was hoping to avoid adding any dev dependencies so the installed bundle is minimal.

Reply:
Yes, dev dependency seems more appropriate. My bad.